config/locales/client.en.yml

```diff
@@ -173,6 +174,8 @@ en:
     select_position: 'Select position'
     statement_placeholder: 'Share your thoughts with the group.'
     aria_statement_label: 'Optionally enter a statement to be recorded with your position'
+    messages:
+      created: Vote added
```

Comment:
Hrmm.. I'm not super keen on this always quote rule. I like brevity and yml has already thought about this.